scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -567,22 +562,22 @@ integrated_sce
 
 There are couple pieces of information here of interest:
 
-- The `corrected` reduced dimension represents the batch-corrected PCA.
-- The `reconstructed` assay represents the batch-corrected normalized expression values, which `fastMNN` "back-calculated" from the batch-corrected PCA.
-Generally speaking, these are not stand-alone values that you should use for other applications like differential gene expression.
-If the `subset.row` argument was provided (as we did here), only genes present in `subset.row` are included in these reconstructed expression values, but this can be overridden so that all genes have reconstructed expression with the argument `correct.all = TRUE`.
+- The `corrected` reduced dimension represents the batch-corrected PCA that `fastMNN` calculated from the `logcounts` assay in the `merged_sce`. 
```

Comment:
We probably want to clarify that this is _not_ the `multiBatchPCA` result, which I think saying it comes from the `logcounts` makes it sound like. 

Maybe we can describe multiBatchPCA as "batch-weighted", whereas this one is actually "batch-corrected"?


```suggestion
- The `corrected` reduced dimension represents the batch-corrected PCA that `fastMNN` calculated. 
```